config/routes.rb

```diff
@@ -6,6 +6,8 @@
         get 'find' => "searches#show"
         get 'find_all' => "searches#index"
         get 'random' => "searches#random"
+        get '/:id/items' => "item_searches#merchant_items"
+        get '/:id/invoices' => "invoice_searches#merchant_invoices"
```

Comment:
try to keep actions down to only show or index, and alter controller names as needed to gain clarification into what its doing. I think the controller names are good in these cases

Reply:
its been a sec but I think restful protocol would be /merchants/:id/items and so forth

Reply:
I see what you mean, so something like

`get '/:id/items' => "item_searches#index
  get '/:id/invoices' => "invoice_searches#index" `